configs/template.env.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
here the the folder name should be named like 'templateLoginData' instead of 'configs'?
 -> templateLoginData/template.env.json